src/main/kotlin/com/exactpro/th2/codec/AbstractSyncCodec.kt

```diff
@@ -105,8 +96,12 @@ abstract class AbstractSyncCodec(
         index: Int,
         group: MessageGroup
     ): MessageGroup? {
+        if (group.messagesCount <= 0)
+            return null
```

Comment:
> group.messagesCount <= 0

Message count cannot be less than zero :)

```suggestion
        if (group.messagesCount == 0) return null
```